luup files/L_RGBController1.lua

```diff
@@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
 	end
 end
 
+-- MySensor RGB(W)
+RGBDeviceTypes["MYS-RGBW"] = {
+	getParameters = function (lul_device)
+		return {
+			name = "MySensors RGBW",
+			settings = {
+				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
+				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
+			}
+		}
+	end,
+
+	getColorChannelNames = function (lul_device)
+		return {"red", "green", "blue", "warmWhite", "coolWhite"}
```

Comment:
Hi vosmont,

Today i finally got time to test your modification.
Sorry, but no success.
First of all on line 1420 the initialParams is missing causing a parse
error.
Secondly the RGB-startup is called  and is initialised but the Arduino
start up is not called so there is no connection with the Arduino gateway
Third since the Arduino plug in handles the actions for it's child's the
RGB code is not executed, id was not able to find a workaround for this.

So i think we should go back to my original idea, create a "parent" (no
child) device by hand with your plug in this works perfect on my system.
and remove the created RGB controller by Arduino or use a dummy

Cheers,
   Bart


On Thu, Oct 15, 2015 at 11:50 PM, vosmont <notifications@github.com> wrote:

> In luup files/L_RGBController1.lua
> <https://github.com/vosmont/Vera-Plugin-RGBController/pull/1#discussion_r42185416>
> :
>
> > @@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
> >  	end
> >  end
> >
> > +-- MySensor RGB(W)
> > +RGBDeviceTypes["MYS-RGBW"] = {
> > +	getParameters = function (lul_device)
> > +		return {
> > +			name = "MySensors RGBW",
> > +			settings = {
> > +				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
> > +				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
> > +			}
> > +		}
> > +	end,
> > +
> > +	getColorChannelNames = function (lul_device)
> > +		return {"red", "green", "blue", "warmWhite", "coolWhite"}
>
> I've created a new branch with your code and some modifications.
> Should (I hope) work from OOB with Arduino plugin.
> As I haven't this plugin, I can not really test.
>
> Please report if this try is successful :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vosmont/Vera-Plugin-RGBController/pull/1/files#r42185416>
> .
>